CONTRIBUTING.md

```diff
@@ -1,14 +1,23 @@
 
+## Validating a Release
+
+* `npm test`
+* `npm run lint`
+* `npm run depcheck`
+* `npm run prettier`
+* `npm run test262`
+* `npm run build`
+
 ## Making a Release
 
-* `npm version patch` (or `major` or `minor`)
-  * that changes `package.json` and `package-lock.json`
+* `npx lerna version patch` (or `major` or `minor`)
```

Comment:
It was suggested that we could have `yarn` commands for chores like this, and have `lerna` installed as a devDependency, so the user/developer wouldn't need to install `lerna` globally. (Using `npx` to run it is certainly better, but maybe a yarn-driven script that used `lerna` internally would be one fewer command to interact with?)